docs/reference/labels-annotations-and-environment-variables.md

```diff
@@ -0,0 +1,46 @@
+| leaderworkerset.sigs.k8s.io/subgroup-key   | Pods that are part of the same subgroup will have the same unique hash value. | 92904e74...801                 | Pod (only if SubGroup is set) |
+
+# Annotations
+
+| Label                                        | Note                                                                 | Example                        | Applies to                   |
+|----------------------------------------------|----------------------------------------------------------------------|--------------------------------|------------------------------|
+| leaderworkerset.sigs.k8s.io/size             | Corresponds to LeaderWorkerSet.Spec.LeaderWorkerTemplate.Size.       | 4                              | Pod                          |
+| leaderworkerset.sigs.k8s.io/replicas         | Corresponds to LeaderWorkerSet.Spec.Replicas.                        | 3                              | Statefulset (only leader)    |
+| leaderworkerset.sigs.k8s.io/leader-name      | The name of the leader pod.                                          | leaderworkerset-multi-template-0 | Pod (only worker)            |
+| leaderworkerset.sigs.k8s.io/exclusive-topology | Specifies the topology for 1:1 exclusive scheduling.                 | cloud.google.com/gke-nodepool  | LeaderWorkerSet, Pod (only if exclusive-topology is used) |
+| leaderworkerset.sigs.k8s.io/subdomainPolicy  | Corresponds to LeaderWorkerSet.Spec.NetworkConfig.SubdomainPolicy.   | UniquePerReplica               | Pod (only if leader and subdomainPolicy set to UniquePerReplica) |
+| leaderworkerset.sigs.k8s.io/subgroup-size      | Corresponds to LeaderWorkerSet.Spec.SubGroupPolicy.SubGroupSize.     | 2                              | Pod (only if SubGroup is set) |
+| leaderworkerset.sigs.k8s.io/subgroup-exclusive-topology | Specifies the topology for 1:1 exclusive scheduling in a given subgroup. | topologyKey                    | LeaderWorkerSet, Pod (only if SubGroup is set and subgroup-exclusive-topology is used) |
+| leaderworkerset.sigs.k8s.io/leader-requests-tpus | Whether the leader pod requests TPU.                                 | true                           | Pod (only if leader pod requests TPU) |
+
+# Environment Variables
+
+| Label            | Note                                                                 | Example                                                                                       | Applies to |
+|------------------|----------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|------------|
+| LwsLeaderAddress | The address of the leader via the headless service.                  | leaderworkerset-multi-template-0.leaderworkerset-multi-template.default                       | Pod        |
```

Comment:
Shouldn't we set this to like `LWS_LEADER_ADDRESS`?

Reply:
Thanks 
It has been changed